components/containers/core/common/features.cc

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/containers/core/common/features.h"
+
+namespace containers::features {
+
+BASE_FEATURE(kBraveContainers,
+             "BraveContainers",
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
We should probably start with DISABLED_BY_DEFAULT and enable slowly via Griffin.